src/MutableContainerInterface.php

```diff
@@ -15,6 +15,8 @@
 {
     public function add(ItemInterface $item): void;
 
+    public static function getIdentifier(ItemInterface $item): string;
```

Comment:
it's pointless to force implementation of this method as you always lock yourself to `MutableContainer`